lib/elasticband/query/score_function/score_mode.rb

```diff
@@ -0,0 +1,21 @@
+module Elasticband
+  class Query
+    class ScoreFunction
+      class ScoreMode < ScoreFunction
```

Comment:
Kind of. They differ on ```MODES``` and on the ```key_name```. I'm following the more verbose standard established on the gem, so that every class responds to a ```to_h``` method. I could, obviously, do a trick to inherit the behaviour, and do some magic, but I think that this will break the standard here.